benchmarks/templates/benchmarks/compare.html

```diff
@@ -2,33 +2,107 @@
+                        {% endfor %}
+                    </select>
+                </div>
             </div>
-            <div class="ylabel-container">
-                <select id="ylabel">
-                    {% for benchmark in benchmarks %}
-                        <option value="{{ benchmark.identifier }}">{{ benchmark.short_name|simplify_domain }}</option>
-                    {% endfor %}
-                </select>
+
+            <div id="comparison-scatter"></div>
+            <div class="download-container columns">
+                <div class="column">
+                    <button id="downloadSVGButton" class="button">Download plot as SVG</button>
+                </div>
+                <div class="column is-narrow"></div>
+                <div class="column has-text-right">
+                    <button id="downloadCSVButton" class="button">Download data as CSV</button>
+                </div>
             </div>
         </div>
 
-        <div id="comparison-scatter"></div>
+        <div class="column">
+            <p>1 dot = 1 model. Only including models where all scores are computed.</p>
+            <br/>
+
+            <h6 class="subtitle subtitle-is-6">Examples of relationships discovered in the literature:</h6>
+            <div class="comparison_selector"
+                 data-benchmark-x="ImageNet-top1_v1" data-benchmark-y="MajajHong2015.IT-pls_v3">
+                <p class="" data-identifier="comparison_selector-imagenet_average">
+                    Object classification performance is correlated with brain alignment.
+                </p>
+                <div data-parent="comparison_selector-imagenet_average">
+                    Researchers found that the performance of computational models in
+                    classifying the object in an image (aka the popular Computer Vision ImageNet benchmark)
+                    is correlated with the alignment of the model to brain and behavioral data.
+                    This is true across different stages of the ventral stream
+                    and has been covered in several publications (e.g.,
+                    <a href="https://www.pnas.org/doi/10.1073/pnas.1403112111">Yamins & Hong et al. 2014</a> with a
+                    smaller image dataset,
+                    <a href="https://www.biorxiv.org/content/10.1101/407007">Schrimpf & Kubilius et al. 2018</a> with
+                    ImageNet).
+                    This connection between machine learning and neuroscience desiderata implies a striking
+                    synergy between research in these two domains, as building better-performing models might also
+                    lead to better models of the brain.
+                </div>
+            </div>
+            <div class="comparison_selector"
+                 data-benchmark-x="FreemanZiemba2013.V1-pls_v2" data-benchmark-y="ImageNet-C-top1_v1">
+                <p class="" data-identifier="comparison_selector-V1_robustness">
```

Comment:
`ImageNet-C-top1_v1` does not appear to be auto-selected when "Alignment to early visual cortex V1 is correlated with model robustness" is selected.

<img width="1739" alt="image" src="https://github.com/user-attachments/assets/c338eac1-7166-42c4-be62-e7a40616b15b">

Reply:
Odd, this does show up on my end. Should we still merge or do you want to comment that button out for now?